apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/api/ApolloCacheHeaders.kt

```diff
@@ -13,4 +16,14 @@ object ApolloCacheHeaders {
    * Records from this request should be evicted after being read.
    */
   const val EVICT_AFTER_READ = "evict-after-read"
+
+  /**
+   * To configure whether to store partial responses, please use the storePartialResponses() extension instead.
+   */
+  @Deprecated(
+      message = "Used for backward compatibility with 2.x",
+      level = DeprecationLevel.ERROR
+  )
+  @ApolloDeprecatedSince(Version.v3_0_1)
+  const val STORE_PARTIAL_RESPONSES: String = ""
```

Comment:
Wanted to add a getter that throws at first, but that would make it less obvious for Java users, as it compiles as a method.